giraffe/src/components/SimpleTable/Types.tsx

```diff
@@ -0,0 +1,41 @@
+export enum ComponentColor {
+    Default = 'default',
+    Primary = 'primary',
+    Secondary = 'secondary',
+    Tertiary = 'tertiary',
+    Success = 'success',
+    Warning = 'warning',
+    Danger = 'danger',
+  }
+  
+  export enum Alignment {
+    Left = 'left',
+    Center = 'center',
+    Right = 'right',
+  }
+  
+  export enum VerticalAlignment {
+    Top = 'top',
+    Middle = 'middle',
+    Bottom = 'bottom',
+  }
+  
+  export enum ButtonShape {
+    Default = 'none',
+    Square = 'square',
+    StretchToFit = 'stretch',
+  }
+  
+  export enum ButtonType {
+    Button = 'button',
+    Submit = 'submit',
+  }
+  
+  export enum BorderType {
+    None = 'none',
+    Horizontal = 'horizontal',
+    Vertical = 'vertical',
+    Both = 'both',
+    All = 'all',
+  }
+  
```

Comment:
since there isn't any JSX in this file, it shouldn't be a .tsx file; just a .ts file.

you'll want to rename it to:

`types.ts`

types should be lowercase and it should have a .ts extension.